lib/curl_setup.h

```diff
@@ -784,10 +780,6 @@
 #  endif
 #endif
 
-#ifdef CURL_WANTS_CA_BUNDLE_ENV
-#error "No longer supported. Set CURLOPT_CAINFO at runtime instead."
```

Comment:
IMO this should not be removed, it was added [intentionally](https://github.com/curl/curl/pull/1542#issuecomment-306118863) to signal to users that the feature is no longer supported in case they are defining it on their own

Reply:
It was never documented as a user option, and the last time it was
implicitly set by a curl build script was 7.5 years ago. Isn't this
a reasonable amount of time for someone setting it explicitly to know
that it's a no-op?

Reply:
I can't believe that was 7 years ago. I didn't notice the date. Ok.